templates/spring-boot-maven-mysql/files/.platform.template.yaml

```diff
@@ -23,7 +23,7 @@ info:
   # The human-readable name of the template.
   name: Spring Boot, Maven, Mysql
```

Comment:
It occurs to me that we should start differentiating MySQL and MariaDB more now that we offer both.  Which means... this template should either switch to use Oracle-MySQL, or it should be renamed and the labeling specify MariaDB.

I'm good with either.  Possibly off topic for this PR, though.